Simplify SUBSCRIBE as a follow-up to FETCH (#510)
moq-wg/moq-transport: draft-ietf-moq-transport.md

A subscription causes the publisher to send newly published objects for a track.
A subscriber MUST NOT make multiple active subscriptions for a track within a
single session and publishers SHOULD treat this as a protocol violation.
The only objects prior to the current object that can be requested are those
in the current group, via the 'Latest Group' filter.

**Filter Types**

The subscriber specifies a filter on the subscription to allow
the publisher to identify which objects need to be delivered.

> 

Author: ```suggestion
```

the subscription starts with the first published or received group.

AbsoluteStart (0x3):  Specifies an open-ended subscription beginning
from the object identified in the StartGroup and StartObject fields. If the
StartGroup is prior to the current group, the publisher SHOULD reply with a
SUBSCRIBE_ERROR with code 'Invalid Range'.

AbsoluteRange (0x4):  Specifies a closed subscription starting at StartObject
in StartGroup and ending at EndObject in EndGroup.  The start and end of the
range are inclusive.  EndGroup and EndObject MUST specify the same or a later

> object than StartGroup and StartObject. If the end has already been published,
> the publisher SHOULD reply with a SUBSCRIBE_ERROR with code 'Invalid Range'.

Author: ```suggestion
object than StartGroup and StartObject. If the StartGroup is prior
to the current group, the publisher SHOULD reply with a
SUBSCRIBE_ERROR with code 'Invalid Range'.
```
